fleet_operations/models/fleet_extended.py

```diff
@@ -504,18 +514,16 @@ class ColorHistory(models.Model):
 
     @api.multi
     def copy(self, default=None):
+        """Method copy."""
         if not default:
             default = {}
         raise Warning(_('You can\'t duplicate record!'))
         return super(ColorHistory, self).copy(default=default)
```

Comment:
@ChavdaRiddhi Test and improve copy method.
I mean no need raise exception we can restrict using the Access right.